sympy/physics/control/lti.py

```diff
@@ -249,6 +252,115 @@ def expand(self):
         """
         return TransferFunction(expand(self.num), expand(self.den), self.var)
 
+    def dcgain(self):
+        """
+        Computes the low-frequency gain of the frequency response in the
+        limit as frequency approaches to zero. For systems with pure
+        integrator terms, DC gain is infinite.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a, b
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction(s + 3, s**2 - 9, s)
+        >>> tf1.dcgain()
+        -1/3
+        >>> tf2 = TransferFunction(p**2, p - 3 + p**3, p)
+        >>> tf2.dcgain()
+        0
+        >>> tf3 = TransferFunction(a*p**2 - b, s + b, s)
+        >>> tf3.dcgain()
+        (a*p**2 - b)/b
+
+        """
+        m = Mul(self.num, Pow(self.den, -1, evaluate=False), evaluate=False)
+        return limit(m, self.var, 0)
+
+    def pole(self, *, multiple=True):
+        """
+        Returns the poles of a transfer function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction((p + 3)*(p - 1), (p - 1)*(p + 5), p)
+        >>> tf1.pole()
+        [-5, 1]
+        >>> tf2 = TransferFunction((1 - s)**2, (s**2 + 1)**2, s)
+        >>> tf2.pole()
+        [I, I, -I, -I]
+        >>> tf3 = TransferFunction(s**2, a*s + p, s)
+        >>> tf3.pole()
+        [-p/a]
+
+        """
+        chareq = Poly(self.den, self.var)
+        chareqroots = roots(chareq, self.var, multiple=True)
+        if len(chareqroots) != degree(chareq):
+            try:
+                chareqroots = [rootof(chareq, k) for k in range(chareq.degree())]
+            except NotImplementedError:
+                raise ValueError("Could not compute poles of this transfer function.")
+        return chareqroots
+
+    def zero(self, *, multiple=True):
+        """
+        Returns the zeros of a transfer function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, a
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> tf1 = TransferFunction((p + 3)*(p - 1), (p - 1)*(p + 5), p)
+        >>> tf1.zero()
+        [-3, 1]
+        >>> tf2 = TransferFunction((1 - s)**2, (s**2 + 1)**2, s)
+        >>> tf2.zero()
+        [1, 1]
+        >>> tf3 = TransferFunction(s**2, a*s + p, s)
+        >>> tf3.zero()
+        [0, 0]
+
+        """
+        chareq = Poly(self.num, self.var)
+        chareqroots = roots(chareq, self.var, multiple=True)
+        if len(chareqroots) != degree(chareq):
+            try:
+                chareqroots = [rootof(chareq, k) for k in range(chareq.degree())]
```

Comment:
I think this would be worth a helper function,
```python
def _roots(poly, var):
    """ like roots, but work on high-order polynomials """
    r = roots(poly, var, multiple=True)
    n = degree(poly)
    if len(r) != degree(poly):
        r = [rootof(poly, var, k) for k in range(n)]
    return r
```